sui/operators.js

```diff
@@ -0,0 +1,301 @@
+    const [operatorAddress] = args;
+
+    const operatorsObjectId = operatorsConfig.objects.Operators;
+    const ownerCapObjectId = options.ownerCapId || operatorsConfig.objects.OwnerCap;
+
+    const tx = new Transaction();
+
+    tx.moveCall({
+        target: `${operatorsConfig.address}::operators::remove_operator`,
+        arguments: [tx.object(operatorsObjectId), tx.object(ownerCapObjectId), tx.pure.address(operatorAddress)],
+    });
+
+    const receipt = await broadcast(client, keypair, tx);
+
+    printInfo('Operator Removed', receipt.digest);
+}
+
+async function mainProcessor(processor, args, options) {
+    const config = loadConfig(options.env);
+
+    if (!config.sui.contracts.Operators) {
+        throw new Error('Operators package not found.');
+    }
+
+    const [keypair, client] = getWallet(config.sui, options);
+    await printWalletInfo(keypair, client, config.sui, options);
+
+    await processor(keypair, client, config, config.sui.contracts.Operators, args, options);
+}
+
+if (require.main === module) {
+    const program = new Command('operators');
+
+    program.description('Operators contract operations.');
+
+    const addCmd = new Command('add')
+        .command('add <newOperatorAddress>')
+        .description('Add an operator')
+        .addOption(new Option('--ownerCap <ownerCapId>', 'ID of the owner capability'))
+        .action((newOperatorAddress, options) => mainProcessor(addOperator, [newOperatorAddress], options));
+
+    const removeCmd = new Command('remove')
+        .command('remove <operatorAddress>')
+        .description('Remove an operator')
+        .addOption(new Option('--ownerCap <ownerCapId>', 'ID of the owner capability'))
+        .action((operatorAddress, options) => mainProcessor(removeOperator, [operatorAddress], options));
+
+    const collectGasCmd = new Command('collectGas')
+        .command('collectGas')
+        .description('Collect gas from the gas service')
+        .addOption(new Option('--receiver <receiver>', 'Address of the receiver'))
+        .requiredOption('--amount <amount>', 'Amount to add gas', parseSuiUnitAmount)
+        .action((options) => mainProcessor(collectGas, [options.amount], options));
+
+    const storeCapCmd = new Command('storeCap')
+        .command('storeCap')
+        .description('Store a capability')
+        .addOption(new Option('--capId <capId>', 'ID of the capability to store'))
+        .action((options) => mainProcessor(storeCap, [], options));
+
+    const removeCapCmd = new Command('removeCap')
+        .command('removeCap <capId>')
+        .description('Remove a capability')
+        .addOption(new Option('--ownerCap <ownerCapId>', 'ID of the owner capability'))
+        .addOption(new Option('--receiver <receiver>', 'The removed cap receiver address'))
+        .action((capId, options) => mainProcessor(removeCap, [capId], options));
+
+    const refundCmd = new Command('refund')
```

Comment:
```suggestion
    const refundCmd = program.addCommand(new Command('refund')
```
Wrap each of this inside an addCommand to avoid the statements below